### DISCUSSION TITLE
Code for IBM i will not function correctly until the temporary library has been corrected in the settings.

### DISCUSSION THREAD

**Discussion by gcraig88keys** [codefori/vscode-ibmi#2201] [General]
Code for IBM i is at V2.12.0.  Since taking this version I'm experiencing these 2 errors.   Temporary library is set to ILEDITOR, and this library exists on the IBM i.    I have also attached the Code for IBM i output log.   Any ideas? 
![image](https://github.com/user-attachments/assets/2b8abf68-57f5-4b0b-9155-e507342a2309)
![image](https://github.com/user-attachments/assets/e2de2f2c-8b27-4a41-a0c4-06ffeb172d8b)
[Code for i Output Log.txt](https://github.com/user-attachments/files/16413225/Code.for.i.Output.Log.txt)

**Comment by worksofliam** [comment]
So if you revert to 2.11.0 it works?

It looks like this is failing. Exit code nine is odd.

```
/home/OCGAR02: /QOpenSys/usr/bin/qsh
system  "CRTLIB LIB(ILEDITOR) TEXT('Code for i temporary objects. May be cleared.')"
{
    "code": 9,
    "signal": null,
    "stdout": "",
    "stderr": ""
}

```

**Reply by sebjulliand** [reply]
And iconv complains that the converter cannot be opened, some other commands return a code 1 and the DB2 output is empty. I'd blame it on `sshd` not being started correctly.
@gcraig88keys I'd advise you take a look at this: https://codefori.github.io/docs/tips/setup/#no-results-from-sql-execution
